packages/paste-core/components/modal/src/Modal.tsx

```diff
@@ -81,7 +81,7 @@ const getAnimationStates = (): any => ({
   from: {opacity: 0, transform: `scale(0.675)`},
   enter: {opacity: 1, transform: `scale(1)`},
   leave: {opacity: 0, transform: `scale(0.675)`},
-  // https://www.react-spring.io/docs/hooks/api
+  // https://www.react-spring.dev/docs/advanced/config
```

Comment:
Question: is it worth adding changesets just for these comment updates? Shall we just ignore danger?

Reply:
Yeah we should ignore danger since we don't publish src code, and our dist files strip comments. So this is just for us in this instance.